Shared/Realm.Shared/RealmList.cs

```diff
@@ -170,42 +80,22 @@ public int Count
 
         #region implementing IList members
 
-        /// <summary>
-        /// Makes a relationship to an item, appending it at the end of the sorted relationship.
-        /// </summary>
-        /// <param name="item">RealmObject being added to the relationship.</param>
```

Comment:
I suspect you have removed comments because things are now buried as non-public methods.

However, have we just **lost** details which are useful to explain our API to users? Is anything replacing these?